From 28d6901c72cb956f1fb13c9b6dd50691f1ccfb49 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 11 Mar 2025 11:41:34 +0000
Subject: [PATCH] Update common submodule

  Richard W.M. Jones (1):
      mlcustomize: Remove dnf --verbose option

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2351282

Cherry picked from commit bb3d0e5bc5f08300ce36bcccf91d02d9d7f32343
and modified to only include dnf --verbose fix.
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common ee88791e1..31d279a79:
diff --git a/common/mlcustomize/guest_packages.ml b/common/mlcustomize/guest_packages.ml
index 2602fc718..96614b6cc 100644
--- a/common/mlcustomize/guest_packages.ml
+++ b/common/mlcustomize/guest_packages.ml
@@ -61,10 +61,7 @@ let install_command packages package_management =
       apt-get $apt_opts update
       apt-get $apt_opts install %s
     " quoted_args
-  | "dnf" ->
-     sprintf "dnf%s -y install %s"
-             (if verbose () then " --verbose" else "")
-             quoted_args
+  | "dnf" ->    sprintf "dnf -y install %s" quoted_args
   | "pisi" ->   sprintf "pisi it %s" quoted_args
   | "pacman" -> sprintf "pacman -S --noconfirm %s" quoted_args
   | "urpmi" ->  sprintf "urpmi %s" quoted_args
